src/pineko/theory.py

```diff
@@ -364,6 +364,13 @@ def fk(self, name, grid_path, tcard, pdf):
                 rich.print(f"Skipping existing FK Table {fk_filename}")
                 return
         max_as = 1 + int(tcard["PTO"])
+        # Check if we are computing FONLL-B fktable and eventually change max_as
+        pine_grid = pineappl.grid.Grid.read(grid_path)
```

Comment:
If you wish `grid` is even enough:
```suggestion
        grid = pineappl.grid.Grid.read(grid_path)
```

Minimal, and non-ambiguous (a grid is by definition a PineAPPL object)